Makefile

```diff
@@ -408,3 +408,8 @@ connect-test-env:
 .PHONY: pre-commit-install
 pre-commit-install:
 	pre-commit install
+
+# Watch the frontend lib for changes and run yarn buildLib whenever a change is detected
+.PHONY: watch-fe-lib
```

Comment:
Let's call this `watch-frontend-lib` for clarity - I think the extra verbosity is fine here